### ISSUE TITLE
Suppress "Error exists after running preLaunchTask" dialog

### ISSUE THREAD

**Issue by condemil** [microsoft/vscode#68084]
I am using vscode with C#, for now when build fails it will show the dialog "Error exists after running preLaunchTask 'build'." with the following options:

* Open launch.json
* Show Errors
* Cancel
* Debug Anyway

It will be great if I will be able to specify to always "Show Errors" in `launch.json`, this way the dialog is not gonna pop up every time build fails (like it is in Visual Studio 2017).

**Comment by supagu** [comment]
Yes! Id rather a popup/toast notification for this.

**Comment by isidorn** [comment]
Yeah, we should provide more fine grained options. Assigning to september so we tackle this some time in the future.

**Comment by isidorn** [comment]
I have changed the dialog such that it now has a checkmark to remember the choice.
If the checkmark is checked the following setting will be written to user settings:
`"debug.onTaskErrors"` it can have three values, the default `prompt`, `debugAnways` and `showErrors`

We have decided against writing this in workspace storage and launch configuration because we never write automatically in lanuch configurations and this feels more like a task thing. The downside of writing into workspace sotrage is that it is hard for hte user to revert this behavior.

Try it out in vscode insiders and let us know what you think. Thanks

![Screenshot 2019-09-05 at 15 49 59](https://user-images.githubusercontent.com/1926584/64348068-d724e500-cff4-11e9-8092-800bcb1842d8.png)
